math/find_smallest_x_and_y_in_algebra.cpp

```diff
@@ -0,0 +1,84 @@
+/**
+ * To find the smallest values of x and y
+ * that satisfy ax - by = 0
```

Comment:
Provide a detailed description, author (in this case, you), and a Wikipedia link in the brief description.